style.css

```diff
@@ -462,12 +498,396 @@ textarea {
+  margin-left: 20px;
+  width: 150px;
+  text-align: center;
+}
+
+.about-butt:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  box-sizing: border-box;
+  color: #f55800;
+}
+
 .footer-social {
   flex: 0.5;
 }
 
+.separation-long {
+  display: none;
+}
+
 .shape {
   text-align: center;
   padding-top: 15px;
-  padding-bottom: 10px;
+  padding-bottom: 5px;
+}
+
+hr.new {
+  border-top: 1px solid #dfe1e6;
+}
+
+@media screen and (max-width: 1400px) and (min-width: 992px) {
+  html,
+  body {
+    min-height: 100%;
+    min-width: 100%;
+  }
+
+  .card-1 {
+    background: url("images/first-work.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  .status-bar {
+    display: none;
+  }
+
+  .toolbar-icon {
+    display: none;
+  }
+
+  .content .top-content {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .intro {
+    position: absolute;
+    left: 6%;
+    top: 20%;
+    padding-left: 30px;
+  }
+
+  .project-details {
+    display: grid;
+    padding-top: 5px;
+  }
+
+  .project-name {
+    display: grid;
+    grid-template-columns: 350px;
+  }
+
+  .project-details p {
+    display: grid;
+    grid-template-columns: 370px;
+  }
+
+  .content {
+    background-image: url("images/top-back.svg");
+    background-repeat: no-repeat;
+    background-position: right -1em top -250px;
+  }
+
+  .title {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 72px;
+    line-height: 81px;
+    color: #f55800;
+  }
+
+  .subtitle {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .illustration {
+    padding-right: 30px;
+  }
+
+  .description {
+    display: grid;
+    grid-template-columns: 500px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .social-media ul {
+    display: inline;
+    position: absolute;
+    left: 2%;
+    top: 30%;
+    padding-right: 60px;
+  }
+
+  .card-single {
+    display: flex;
+  }
+
+  .head {
+    border-bottom: none;
+  }
+
+  .main-butt {
+    padding-left: 20px;
+  }
+
+  .about-dec {
+    display: flex;
+    padding-left: 100px;
+    padding-bottom: 60px;
+  }
+
+  .border {
+    display: none;
+  }
+
+  .titl {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 72px;
+    line-height: 81px;
+    padding-left: 30px;
+    text-align: start;
+  }
+
+  .about-me {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    margin-left: 25px;
+  }
+
+  .single-work img {
+    width: 600px;
+  }
+
+  .all-projects {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 20px;
+    justify-content: center;
+    padding-top: 30px;
+    padding-left: 200px;
+    align-items: center;
+  }
+
+  .about-ill {
+    padding-top: 120px;
+    align-items: center;
+  }
+
+  .contact {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
```

Comment:
- [ ] There is an overlap between the `works` section and the `about` section. Kindly fix that.
- [ ]  Kindly stick to the Figma design as much as you can, the left and the right margins of the `about` section are not the same as the margins provided in the Figma design. Kindly fix that.
<img width="433" alt="image" src="https://user-images.githubusercontent.com/77190203/154686997-12bf5f58-9248-4b30-a3f1-c1573accda0c.png">

- [ ] There is a large gap at the bottom of the `about` section, Kindly edit your design to match the Figma design:
<img width="428" alt="image" src="https://user-images.githubusercontent.com/77190203/154687585-f9cb599f-4f2e-4691-a4b6-d58b015cfb32.png">